lib/webauthn_components/base_components.ex

```diff
@@ -13,7 +13,7 @@ defmodule WebauthnComponents.BaseComponents do
   @doc false
   def button(assigns) do
     ~H"""
-    <button id={@id} type={@type} {@rest}>
+    <button id={@id} type={@type} {@rest} {if @disabled, do: %{disabled: true}, else: %{}}>
```

Comment:
This syntax accomplishes the same result, and it's more idiomatic for component assigns.

```suggestion
    <button id={@id} type={@type} disabled={@disabled} {@rest}>
```

Reply:
I pushed dcd6613 for this.

Reply:
This works, which is surprising for me.

I know `disabled` matches CSS selectors simply by being present on the element.

However, the attribute is simply not present for `disabled={false}` (at least, on my version of LiveView), which is fantastic news.